pnpm-lock.yaml

```diff
@@ -222,8 +222,8 @@ importers:
       '@lukeed/uuid': 2.0.0
       cookie: 0.4.1
     devDependencies:
-      '@sveltejs/adapter-auto': 1.0.0-next.5
-      '@sveltejs/kit': 1.0.0-next.203_svelte@3.44.2
+      '@sveltejs/adapter-auto': link:../../../adapter-auto
```

Comment:
What's this change? This looks fishy.

Reply:
idk, i've noticed that this sort of thing just happens from time to time